src/cryptoadvance/specter/wallet.py

```diff
@@ -1153,8 +1151,6 @@ def keypoolrefill(self, start, end=None, change=False):
         # Descriptor wallets were introduced in v0.21.0, but upgraded nodes may
         # still have legacy wallets. Use getwalletinfo to check the wallet type.
         # The "keypool" for descriptor wallets is automatically refilled
```

Comment:
This comment is specific to the code which just got deleted, right? Please adjust the comment as well.